src/main/resources/messages.properties

```diff
@@ -42,4 +38,14 @@ too.big.email=Email length should be less than 129 characters. Please enter vali
 
 valid.email.regexp=^[\\w-\\+]+(\\.[\\w]+)*@[\\w-]+(\\.[\\w]+)*(\\.[a-zA-Z]{2,})$
 valid.password.regexp=(?=.*?[0-9])(?=.*?[a-z])(?=.*?[A-Z])(?=\\S+$).+.{8,30}
-valid.username.regexp=(?=\\S+$).{2,50}
\ No newline at end of file
+valid.username.regexp=(?=\\S+$).{2,50}
+
+## Exceptions messages
+user.not-found.exception=User not found
+security.exception=Security breach
+illegal.id.exception=Identifier does not match
+invalid.dto.exception=Invalid data transfer object
+sql.exception=Exception during saving object to the database
+entity.not-found.exception=Entity not found into the database
+illegal.argument.exception=Exception during request. Illegal or inappropriate argument
+validation.exception=Validation exception
```

Comment:
Make these parameter more consistent - move word "exception" forward:
**exception.user.not-found;
exception.illegal.id.**

Reply:
Performed